docs/COMMUNICATION.rst

```diff
@@ -37,4 +37,10 @@ special events. The regularly scheduled meetings include a community
 meeting to discuss code development, a project meeting to discuss
 broader aspects of the PlasmaPy project like education and outreach to
 the broader plasma community, and meetings of PlasmaPy working groups.
-The special events include `Plasma Hack Week`_.
+The special events include |Plasma Hack Week|.
+
+.. _GitHub Discussions page: https://github.com/PlasmaPy/PlasmaPy/discussions
+.. _Gitter bridge: https://gitter.im/PlasmaPy/Lobby
+.. _mailing list: https://groups.google.com/forum/#!forum/plasmapy
+.. _PlasmaPy meetings: https://www.plasmapy.org/meetings
+.. _suggestion box: https://docs.google.com/forms/d/e/1FAIpQLSdT3O5iHZrLJRuavFyzoR23PGy0Prfzx2SQOcwJGWtvHyT2lw/viewform?usp=sf_link
```

Comment:
For situations like this where the links/substitutions were only used on one page, I moved the definitions to that page rather than include them in the global substitutions.